operating_procedures.md

```diff
@@ -154,283 +111,73 @@ FOCUS Operating Procedures
-
-During the Review & Approval process members MAY raise comments or objections.
-
-Comments MUST be taken in consideration by the Working Group, but they MAY be dismissed if they group thinks that are not relevant.
-
-Objections MUST be taken in consideration and they SHALL NOT be dismissed by the Working Group without being reviewed.
-
-If a contribution receives an objection the group MUST resolve the issue, with the person that raised the objection, before deciding the status of the contribution. If the objection is sustained, meaning the person doesn’t remove it, then the group will have to recur to a vote to resolve it.
-
-### 6.1.2 Approval Criteria
-
-A contribution is considered approved and therefore it can be merged if:
-The contribution has not received any sustainable objection during the review period, AND At least 3 reviewers have indicated that they agree with the contribution
-
-If a sustained objection is received, the contribution cannot be merged, even if 3 or more contributors agreed with the contribution.
-
-If during the review period a contribution receives a comment, it is up to the group or maintainer(s) to accept the comment or not. In any case, in order to merge the contribution at least 3 reviewers MUST indicate that they agree with the contribution.
-
-# 7\. FOCUS Process Flows
+# 4\. FOCUS Working Group (FG) 
 
-## 7.1 Work Packages
+Each FOCUS Working group may have an operational_proceedures.md document in the FOCUS Group GitHub respository that outlines their processes and proceedures.
 
-[TBC]
+# 5\. FOCUS Process Flows
 
-## 7.2 Technical Spec Lifecycle
+The workflow for all Working Groups covered by Patent Policy Option 4: W3C Mode will follow the guidelines in [w3c_mode_workflow.md](w3c_mode_workflow.md).
 
-# 8\. Github Flows
+# 6\. Github Flows
 
-## 8.1 GitHub Access Rights
+## 6.1 GitHub Access Rights
 
 Table 8.1-1: GitHub Access Rights
-|     Role     |                                                              Access Rights                                                              |   |
-|:------------:|:---------------------------------------------------------------------------------------------------------------------------------------:|---|
-| Participants | TRIAGE - Can read and clone this repository. Can also manage issues and pull requests.                                                  |   |
-| Editors      | WRITE - Can read, clone, and push to this repository. Can also manage issues and pull requests.                                         |   |
-| Maintainer   | ADMINISTRATOR - Can read, clone, and push to this repository. They can also manage issues, pull requests, and some repository settings. |   |
+|     Role           |                                                              Access Rights                                                                |   |
```

Comment:
You also may want to signal hygiene requirements - "All Members are required to have 2FA enabled" "Files should have appropriate SPDX headers" etc